src/huggingface_hub/repocard.py

```diff
@@ -99,3 +104,162 @@ def metadata_eval_result(
         model_index, dict_factory=lambda x: {k: v for (k, v) in x if v is not None}
     )
     return {"model-index": [data]}
+
+
+def metadata_update(
+    repo_id: str,
+    metadata: Dict,
+    repo_type: str = None,
+    overwrite: bool = False,
+    token: str = None,
+) -> None:
```

Comment:
If taking into account the `return` proposed by Julien:
```suggestion
) -> str:
```